CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,132 @@
+# Contributor Covenant Code of Conduct
```

Comment:
Should we create an organization repository .github and add the file there. Then every project could reference the same Code of conduct at the root in their readme? This way we could centralize then updates of this document